### PR TITLE
feat: serialize the new property isPrimaryKey

### PR DESCRIPTION
**Pull request by ghusse**
Linked to CU-48py0a

## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by arnaudbesnier** [comment]
Task linked: [CU-48py0a Lumber - ID fields with no default value are missing](https://app.clickup.com/t/48py0a)

**Review by DrRaider — APPROVED** [review]

**Review by arnaudbesnier — APPROVED** [review]

File: src/serializers/schema.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ function SchemaSerializer() {
         'enums',
         'integration',
         'isFilterable',
+        'isPrimaryKey',
```
Comment (arnaudbesnier) [inline src/serializers/schema.js:34]: Actually wondering if the `isPrimaryKey` property is serialized to `true` or `false` for any field as it is for other properties (and if not, if it should)

Comment (ghusse) [reply src/serializers/schema.js:34]: @arnaudbesnier I don't understand the question.

Do you want to know if there is always a value set for this property, on every field?

For what I remember, properties like `isRequired` is only set to true when a field is required, and not set at all when a field is not.

I did the same for `isPrimaryKey`, but this behavior is part of both `forest-express-sequelize` and `forest-express-mongoose`, not this package.

Comment (ghusse) [reply src/serializers/schema.js:34]: Ok, I changed the code to always set the property

**Review by arnaudbesnier — APPROVED** [review]

**Pull request merged**